config/fund_loader_config/cof/cof_r2.py

```diff
@@ -100,6 +143,47 @@
             "https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice/"
             "community-ownership-fund-privacy-notice"
         ),
+        "reference_contact_page_over_email": True,
+        "contact_us_banner_json": {
+            "en": textwrap.dedent(
+                """
+                 <h2 class="govuk-heading-m">Get application support</h2>
```

Comment:
@tferns can't we have a same structure of contact us page ?
then we can store related data on fund store and rest of the html content can go to the front end ? may be I am wrong this is just an idea.

Reply:
Yes we could, I think ideally we’d have just a generic contact us page and only provide data to it.  But designs have been introduced specific to COF and this is a more flexible (yet ugly) way of making them configurable